src/query/api/v1/handler/influxdb/write.go

```diff
@@ -243,6 +243,12 @@ func (ii *ingestIterator) Error() error {
 	return ii.err.FinalError()
 }
 
+func (*ingestIterator) SetCurrentMetadata(_ ts.Metadata) {}
+
+func (*ingestIterator) CurrentMetadata() ts.Metadata {
+	return ts.Metadata{}
```

Comment:
May as well implement this as well? It would be weird if we start using this and for some reason the drop rules don't apply just to influx?